workplace/Living in Open Workspaces.md

```diff
@@ -0,0 +1,80 @@
+## Living in Open Workspaces
+
+Total time: 35 minutes
+
+- Lesson: 15 min
+- Check for Understanding: 20 min
+
+## Motivation
+
+In a [recent study](https://www.business2community.com/human-resources/how-to-design-a-workspace-that-boosts-employee-motivation-02222288), research company IPSOS, discovered that a workspace focused on employee happiness is a powerful driver for productivity.
+
+The IPSOS study showcases the urgent need for more engaging workspaces:
+
+- 85% of workers feel they don’t work in a space where they can share ideas
+- 87% feel there isn’t a place to work in teams
+- 84% don’t feel a sense of belonging or a willingness to be a part of company culture
+
+## Objectives
+
+Participants will be able to:
+
+- Know in detail about workspace management
+- Manage themselves properly in work environment
+- Learn the skills to be a leader in the workplace
```

Comment:
How do they learn this?  Would you tell me how you think leadership comes into a "Living in Open Workspaces" lesson?